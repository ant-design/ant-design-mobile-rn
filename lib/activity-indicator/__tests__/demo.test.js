import rnDemoTest from '../../../tests/shared/demoTest';

rnDemoTest('activity-indicator');
