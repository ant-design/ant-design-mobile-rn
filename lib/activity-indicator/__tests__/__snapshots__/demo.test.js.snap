// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/activity-indicator/demo/basic.tsx correctly 1`] = `
<View
  style={
    Array [
      Object {
        "marginTop": 20,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "marginLeft": 15,
          "marginRight": 15,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "flexWrap": "nowrap",
            "justifyContent": "flex-start",
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            undefined,
          ]
        }
      >
        <Text>
          Icon without text
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "center",
            }
          }
        >
          <ActivityIndicator
            animating={true}
            color="gray"
            hidesWhenStopped={true}
            size="small"
          />
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "height": 21,
        },
        Object {
          "backgroundColor": "#fff",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "marginLeft": 15,
          "marginRight": 15,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "flexWrap": "nowrap",
            "justifyContent": "flex-start",
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            undefined,
          ]
        }
      >
        <Text>
          Icon with text
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "center",
            }
          }
        >
          <ActivityIndicator
            animating={true}
            color="gray"
            hidesWhenStopped={true}
            size="small"
          />
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 14,
                  "marginLeft": 8,
                },
              ]
            }
          >
            Loading...
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "height": 21,
        },
        Object {
          "backgroundColor": "#fff",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "marginLeft": 15,
          "marginRight": 15,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "flexWrap": "nowrap",
            "justifyContent": "flex-start",
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            undefined,
          ]
        }
      >
        <Text>
          Dark Background
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#2B2F42",
                "height": 89,
                "justifyContent": "center",
                "width": 89,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              }
            }
          >
            <ActivityIndicator
              animating={true}
              color="#fff"
              hidesWhenStopped={true}
              size="small"
            />
          </View>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "height": 21,
        },
        Object {
          "backgroundColor": "#fff",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "marginLeft": 15,
          "marginRight": 15,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "flexWrap": "nowrap",
            "justifyContent": "flex-start",
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            undefined,
          ]
        }
      >
        <Text>
          Large Size
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "center",
            }
          }
        >
          <ActivityIndicator
            animating={true}
            color="gray"
            hidesWhenStopped={true}
            size="large"
          />
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "height": 21,
        },
        Object {
          "backgroundColor": "#fff",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "marginLeft": 15,
          "marginRight": 15,
        },
        undefined,
      ]
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "alignItems": "center",
            "borderRadius": 5,
            "borderWidth": 1,
            "justifyContent": "center",
          },
          Object {
            "height": 47,
            "paddingLeft": 15,
            "paddingRight": 15,
          },
          Object {
            "backgroundColor": "#ffffff",
            "borderColor": "#dddddd",
          },
          false,
          false,
          undefined,
          undefined,
        ]
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "fontSize": 18,
              },
              Object {
                "color": "#000000",
              },
              false,
              false,
            ]
          }
        >
          Click to show Toast
        </Text>
      </View>
    </View>
  </View>
</View>
`;
